src/algebra/order/monoid.lean

```diff
@@ -1001,6 +1001,19 @@ protected lemma add_lt_add_of_lt_of_le [preorder α] [covariant_class α α (+)
   [covariant_class α α (swap (+)) (<)] (hc : c ≠ ⊤) (hab : a < b) (hcd : c ≤ d) : a + c < b + d :=
 (with_top.add_lt_add_right hc hab).trans_le $ add_le_add_left hcd _
 
+/-  It is not `to_additive` of `with_top.map_mul_of_mul_hom`, since `with_top` does not have
+a multiplication. -/
```

Comment:
```suggestion
/-  There is no `with_top.map_mul_of_mul_hom`, since `with_top` does not have
a multiplication. -/
```

Reply:
I used your suggestion, and also added it to the `with_bot` version.